docker/api/client.html

```diff
@@ -0,0 +1,12 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.2.0/socket.io.js" integrity="sha256-yr4fRk/GU1ehYJPAs8P4JlTgu0Hdsp4ZKrx8bDEDC3I=" crossorigin="anonymous"></script>
+<script type="text/javascript" charset="utf-8">
+var socket = io.connect("https://localhost");
+socket.on('connect', function() {
+    console.log('Websocket connected!');
+    var ret = socket.emit('logs', {'level': 'DEBUG'});
+});
+socket.on('cnaas_log', function(data) {
+    console.log(data);
+});
+</script>
+
```

Comment:
Shouldn't it at least be valid HTML? :-)

Reply:
Haha I don't know, I hope no one is ever going to have to use this.. Did I mention you have to open the browser debugger to actually see the output from console.log()? :D
I added some html tags now..